metrics/interface/src/main/java/org/apache/iotdb/metrics/enums/MetricLevel.java

```diff
@@ -17,16 +17,22 @@
  * under the License.
  */
 
-package org.apache.iotdb.metrics.micrometer;
+package org.apache.iotdb.metrics.enums;
 
-import io.micrometer.core.instrument.Meter;
-import io.micrometer.core.instrument.Tags;
+public enum MetricLevel {
+  all(0),
+  normal(1),
+  important(2),
+  core(3);
 
```

Comment:
All fields in enum should be in uppercase

Reply:
this is in lowercase because of the load of yml property

Reply:
This has been fixed.